counsel.el

```diff
@@ -2864,8 +2864,21 @@ INITIAL-DIRECTORY, if non-nil, is used as the root directory for search."
               :preselect (counsel--preselect-file)
               :require-match 'confirm-after-completion
               :history 'file-name-history
+              :keymap counsel-file-jump-map
```

Comment:
The variable is not yet defined here, giving rise to the following byte-compilation warning:
```
In counsel-file-jump:
counsel.el:2867:23: Warning: reference to free variable
    ‘counsel-file-jump-map’
```
The solution is to move the variable's definition before to a point before its first use.

Reply:
Ok, I hope that's fixed now.

Reply:
Yes, thanks.